modules/GenericDataParser/src/server/generic_data_parser.R

```diff
@@ -4628,6 +4628,10 @@ getSubjectAndTreatmentData <- function (precise, genericDataFileDataFrame, calcu
 validateSubjectData <- function(subjectData, dryRun) {
   # Validates Subject Data
   # For now, just passes information to validateValue Kinds
+  if (is.null(subjectData)) {
+    return(NULL)
+  }
+  # subjectData is just list() with the new thing so why is it 
```

Comment:
@brianbolt this comment looks incomplete. Is this intentional?

Reply:
I did not mean to leave this comment in, thanks for noting it.  I removed it.